pkg/quota/admission/runonceduration/api/types.go

```diff
@@ -5,18 +5,19 @@ import (
 )
 
 // RunOnceDurationConfig is the configuration for the RunOnceDuration plugin.
-// It specifies a default override value for ActiveDeadlineSeconds for a run-once pod.
+// It specifies a maximum value for ActiveDeadlineSeconds for a run-once pod.
 // The project that contains the pod may specify a different setting. That setting will
 // take precedence over the one configured for the plugin here.
 type RunOnceDurationConfig struct {
 	unversioned.TypeMeta
 
-	// ActiveDeadlineSecondsOverride is the value to set on containers of run-once pods
+	// ActiveDeadlineSecondsLimit is the maximum value to set on containers of run-once pods
 	// Only a positive value is valid. Absence of a value means that the plugin
 	// won't make any changes to the pod
-	ActiveDeadlineSecondsOverride *int64
+	ActiveDeadlineSecondsLimit *int64
 }
 
-// ActiveDeadlineSecondsOverrideAnnotation can be set on a project to override the number of
+// ActiveDeadlineSecondsLimitAnnotation can be set on a project to limit the number of
 // seconds that a run-once pod can be active in that project
-const ActiveDeadlineSecondsOverrideAnnotation = "openshift.io/active-deadline-seconds-override"
+// TODO: this label needs to change to reflect its function. It's a limit, not an override.
```

Comment:
note in the TODO that we are keeping it this way for compatibility, so someone doesn't just change it as cleanup without considering that